### PR TITLE
L10N-315 2021-06-10-645 - translations batch

### PR DESCRIPTION
**Pull request by alexandermccate**

### CODE REVIEW

**Review by allison-c — APPROVED** [review]

File: packages/L10n/src/locales/de.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ de:
       drop_files_here_or: Legen Sie Dateien zum Hochladen hier ab oder
       choose_from_computer: wählen Sie sie auf Ihrem Computer aus
       choose_from_computer_a11y: Wählen Sie Dateien auf Ihrem Computer aus
-      cancel_upload: Upload abbrechen
-      restart_upload: Wählen Sie die Datei erneut aus, um den Vorgang zu wiederholen
+      cancel_upload: Upload für {{Name}} abbrechen
```
Comment (agherreras) [inline packages/L10n/src/locales/de.yml:22]: should it be {{name}}, with non-capital "N"?

**Review by allison-c — DISMISSED** [review]
yes, they should be lowercase `name`

File: packages/L10n/src/locales/de.yml [hunk 2]
```diff
@@ -19,8 +19,8 @@ de:
...
+      restart_upload: Wählen Sie {{Name}} erneut, um es erneut zu versuchen
```
Comment (danny-dywang) [inline packages/L10n/src/locales/de.yml:23]: This tag is capitalized too.

**Review by danny-dywang — APPROVED** [review]

**Comment by alexandermccate** [comment]
@allison-c please 🆗 
changes applied

**Review by nahumzs — APPROVED** [review]

**Pull request merged**